ooni/errors.py

```diff
@@ -148,6 +149,9 @@ def failureToString(failure):
     elif isinstance(failure.value, ProcessDone):
         string = 'process_done'
 
+    elif isinstance(failure.value, ConnectionDone):
+        string = 'connection_done'
```

Comment:
If you modify this later you probably need to sync [the specification](https://github.com/TheTorProject/ooni-spec/blob/master/data-formats/df-000-base.md#error-strings).